source/usage.rst

```diff
@@ -23,6 +26,18 @@ Command Line Arguments
 
  - The ``.manimdsl`` file to compile
 
+-f, --show_file_in_finder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Flag to show the output file in finder automatically
+ - The Python file will be saved with the same name as the output MP4 file (and so is defaulted to ``out.py``)
```

Comment:
This flag doesn't really have anything to do with the python file naming (that's covered in ---python)

Reply:
updated